menor_num_aleatorio_array.rb

```diff
@@ -0,0 +1,18 @@
+rand_array = []
+
+10.times do |index|
+  rand_array.push(rand(0..100))
+end
+
+puts "#{rand_array}"
+min_number = rand_array[0]
+puts "++++++++++++++#{min_number}" 
+
+rand_array.each_with_index do |value, index|
+  puts "O número #{rand_array[index]} está na posição #{index}"
+  min_number = rand_array[index] if rand_array[index].to_i < min_number
+  puts "Agora o menor número é #{min_number} e está na posição #{index}"
+end
+
+puts "---------------#{min_number}" 
+
```

Comment:
Fala Dani, Boa!
Eu faria um processo diferente:

```suggestion
puts "#{rand_array}"
min_number = rand_array[0]

rand_array.each do |value|
  min_number = value if value < min_number
end

puts "Número menor#{min_number}"

puts "---------------#{min_number}" 

```

O que pensa?

Reply:
Faz total sentido. Não sei porque não usei o `value `